src/app/components/Transactions/ConsensusTransactions.tsx

```diff
@@ -56,73 +56,82 @@ export const ConsensusTransactions: FC<ConsensusTransactionsProps> = ({
         ]
       : []),
   ]
-  const tableRows = transactions?.map(transaction => ({
-    key: `${transaction.hash}${transaction.index}`,
-    data: [
-      {
-        content: <StatusIcon success={transaction.success} error={transaction.error} />,
-        key: 'success',
-      },
-      {
-        content: <TransactionLink scope={transaction} alwaysTrim hash={transaction.hash} />,
-        key: 'hash',
-      },
-      {
-        content: <BlockLink scope={transaction} height={transaction.block} />,
-        key: 'round',
-      },
-      {
-        content: <Age sinceTimestamp={transaction.timestamp} />,
-        key: 'timestamp',
-      },
-      ...(verbose
-        ? [
-            {
-              content: <ConsensusTransactionMethod method={transaction.method} truncate />,
-              key: 'method',
-            },
-            {
-              align: TableCellAlign.Right,
-              content: (
-                <Box
-                  sx={{
-                    display: 'flex',
-                    alignItems: 'center',
-                    position: 'relative',
-                    pr: 3,
-                  }}
-                >
+
+  const tableRows = transactions?.map(transaction => {
+    const targetAddress = transaction.body.to || transaction.body.account
```

Comment:
Transactions consensus and runtime APIs are not consistent:
 
runtime has props
```
amount?: string;
to?: Address;
```
where here we use body. not sure this should be unified in Nexus.